earth_enterprise/src/common/filebundle/cachedreadaccessor_unittest.cpp

```diff
@@ -238,6 +241,8 @@ class CachedReadAccessorUnitTest : public UnitTest<CachedReadAccessorUnitTest> {
     // same block cached read
     offset *= 2;
     request_count -= offset;
+    buffer.resize(request_count);
```

Comment:
There's a potential problem here that I didn't think about when we talked the other day.  Every call to resize could potentially change the location of the underlying character array, which would invalidate out_buffer.  That shouldn't happen since we've already reserved enough capacity in the string, but it's technically possible.  It would be safer to just run buffer.resize(block_size) at the top of the function and remove the extra resize calls.